pkg/utils/klogtologrus/adapter.go

```diff
@@ -14,28 +14,39 @@ import (
 //   import (
 //     _ "github.com/flant/shell-operator/pkg/utils/klogtologrus"
 //   )
-func init() {
+func InitAdapter(enableDebug bool) {
```

Comment:
OK, I've tried to recall the reason for `init` except the "automagic" one... No success. So, let's use explicit initialization for this adapter.